File: pset6/caesar.py
import cs50
import sys

z = 65
y = 97

def main():
    
    if len(sys.argv) != 2:
        print(" Usage: ./caesars k")
        return 1
    
    else:
        print("plaintext: ", end="")
        s = cs50.get_string()
        k = int(sys.argv[1])
    
        print("ciphertext: ", end="")
        
        for c in s:
            if ord(c) >= 65 and ord(c) <= 90:
                print("{}".format(chr((((ord(c)) + k - z) % 26) + z)), end="")
            
            elif ord(c) >= 97 and ord(c) <= 122:
                print("{}".format(chr((((ord(c)) + k - y) % 26) + y)), end="")
            
            else:
                j = c
                print("{}".format(j))
        
    print("")

if __name__ == "__main__":
    main()

File: pset2/caesar.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>
#include <ctype.h>
#include <stdlib.h>


int z = 65;
int y = 97;
string s;
int main (int argc, string argv[])
{

    if (argc == 2)
    {
        printf("plaintext:");
         s = get_string();
         //x = argv[1];
         int k = atoi(argv[1]);
       
       
        printf("ciphertext:");
    
    
        for (int i = 0, n = strlen(s); i < n; i++)
        {
            
        
            //if (s[i] == ' ')
            //{
              //  printf(" ");
            //}
            //if (s[i] == ',')
            //{
             //   printf(",");
           // }
             if (s[i] >= 65 && s[i]<=90)
            {
        
            printf("%c", (((s[i] + k - z) % 26) + z));
            }
            else if (s[i] >= 97 && s[i]<=122)
            {
        
            printf("%c", (((s[i] + k - y) % 26) + y));
            }
            else
            {
                printf("%c", s[i]);
            }
        }
    
        printf("\n");
    }
    
        
    else
    {
        printf("Usage: ./caesars k\n");
        return 1;
    }
}
